x-pack/plugins/security_solution/public/management/pages/policy/view/host_isolation_exceptions/components/empty_non_existent.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiEmptyPrompt, EuiPageTemplate } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+export const PolicyHostIsolationExceptionsEmptyUnexisting = ({
```

Comment:
Should the file name be like the component name?

Reply:
I am personally not a fan of repeating the same string over and over on sub-files:

`policy/view/host_isolation_exceptions/components/empty_non_existent.tsx`

vs

`policy/view/host_isolation_exceptions/components/policy_host_isolation_exceptions_empty_non_existent.tsx`

Do we have a proper guideline or is down to preference?

Reply:
Yeah, I'm partially agree on that. But if the component name is `...EmptyUnexisting` I think the file name should be `empty_unexisting` and not `empty_not_existent` right?

Reply:
ahh! I totally agree there. let me change that.

Reply:
I renamed it and pushed the change.